frappe/public/js/frappe/ui/toolbar/energy_points_notifications.js

```diff
@@ -0,0 +1,158 @@
+			return 'Today';
+		} else if(date > prev_week) {
+			return 'Last 7 days';
+		} else if(date > prev_month) {
+			return 'Last 30 days';
+		} else  {
+			return 'Older';
+		}
+	}
+
+	get_energy_points_list(limit) {
+		return frappe.db.get_list('Energy Point Log', {filters: {user: frappe.session.user, type:['not in',['Review']]},
+			fields:['name','user', 'points', 'reference_doctype', 'reference_name', 'reason', 'type', 'seen', 'rule', 'owner', 'creation'],
+			limit:limit, order_by:'creation desc'}).then((energy_points_list)=> {
+			return energy_points_list;
+		});
+	}
+
+	render_energy_points_dropdown() {
+		let header_html =
+			`<li class="points-updates-header">
+				<span class="points-updates-title bold text-muted">${__('Energy Points')}</span>
+				<a href="#social/users" class="points-leaderboard text-muted hidden-xs">${__('Leaderboard')}</a>
+			</li>`;
+		let body_html = '';
+
+		if(this.dropdown_items.length) {
+			let date_range= this.get_energy_points_date_range(this.dropdown_items[0].creation);
+			body_html += `<li class="points-date-range text-muted">${__(date_range)}</li>`;
+			this.dropdown_items.forEach(field=> {
+				let current_field_date_range = this.get_energy_points_date_range(field.creation);
+				if(date_range !== current_field_date_range) {
+					body_html+=`<li class="points-date-range text-muted">${__(current_field_date_range)}</li>`;
+					date_range=current_field_date_range;
+				}
+				let item_html = this.get_dropdown_item_html(field);
+				if(item_html) body_html += item_html;
+			});
+		} else {
+			body_html += `<li><a href="#" onclick = "return false" class="text-muted text-center">${__('No activity')}</a></li>`;
+		}
+
+		let dropdown_html = header_html + body_html;
+		this.$dropdown_list.html(dropdown_html);
+	}
+
+	get_dropdown_item_html(field) {
+		let doc_link = frappe.utils.get_form_link(field.reference_doctype,field.reference_name);
+		let link_html_string = field.seen? `<a href=${doc_link}>`: `<a href=${doc_link} class="unseen">`;
+		let points_html= field.type === 'Auto' || field.type === 'Appreciation'
+			? `<div class="points-update positive-points">+${__(field.points)}</div>`
+			: `<div class="points-update negative-points">${__(field.points)}</div>`;
+		let message_html = this.get_message_html(field);
+
+		let item_html = `<li class="recent-points-item">
+							${link_html_string}
+							${points_html}
+							<div class="points-reason">
+								${message_html}
+							</div>
+							</a>
+						</li>`;
+		return item_html;
+	}
+
+	get_message_html(field) {
+		let owner_name = frappe.user.full_name(field.owner).trim();
+		owner_name = this.truncate_value(owner_name);
+		let message_html = '';
+		if(field.type === 'Auto' ) {
+			message_html = `For ${__(field.rule)} <span class="points-reason-name text-muted">${__(field.reference_name)}</span>`;
+		} else {
+			let message;
+			if(field.type === 'Appreciation') {
+				message = `${__(owner_name)} appreciated your work on `;
```

Comment:
Need to fix translated strings here...

Please refer: https://frappe.io/docs/user/en/translations#3-blocks 

Suggestion...
```suggestion
				message = __('{0} appreciated your work on {1}', [owner_name, message_html]);
```